packages/components/yarn.lock

```diff
@@ -0,0 +1,12393 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
@didoo Just noticed this, there shouldn't be a yarn.lock in anything in /packages/*. Only at the root. Please remove this file before merge 

We could add to the .gitignore, but not sure if solves the root cause of not having the file generated there at all

Reply:
I've removed the file.